src/tactic/local_cache.lean

```diff
@@ -85,12 +83,12 @@ end block_local
 
 namespace def_local
 
--- Fowler–Noll–Vo hash function (FNV-1a)
+-- Fowler-Noll-Vo hash function (FNV-1a)
 section fnv_a1
 
 def FNV_OFFSET_BASIS := 0xcbf29ce484222325
 def FNV_PRIME := 0x100000001b3
-def RADIX := 0xFFFFFFFFFFFFFFFF + 1 -- 64-bits
+def RADIX := 2^64
```

Comment:
I would really use `apply_normed` here. Otherwise, it will get very slow

Reply:
Sure, I'll do it, but just so I learn something: why? Wouldn't this just be stored in the internal big nat representation from libgmp the first time it's parsed, and so will only every be calculated once (and at compile/load time)?

Reply:
The difference is if we use it in proofs, which we can because it's not meta.